add kate's super easy spag bol recipe (#22)
compsoc-edinburgh/cook-book: main_courses/0-easy/kates_super_easy_spag_bol.md

---
serves: 4
allergens:  [gluten]
---

# Kate's Super Easy Spaghetti Bolognese

Ridiculously simple spaghetti bolognese recipe; not super healthy, but it'll do in a pinch. Easy to scale up by 1.5x for 6 servings. 

## Ingredients

 - 500g beef mince
 - 1 tbsp olive oil
 - 2 tins chopped tomatoes
 - 2 tbsp tomato purée
 - 1 stock cube
 - seasoning (onion granules suggested)
 - 400g spaghetti

## Method

1. Heat up a pan with olive oil. Keep the heat at medium throughout. 
2. Add the beef mince and brown, mixing the mince until fully browned. 
3. Add the chopped tomatoes, tomato purée, and seasoning while stirring. 

> 4. Make up around 400ml of stock with the stock cube, then add to the pan. 

Reviewer: ```suggestion
4. Dissolve the stock cube in around 400ml of hot water, then add to the pan.
```

5. Stir in fully and put a lid on the pan. 

> 6. Simmer for around 45 minutes or until thick. Make sure the temperature is consistent and it won't boil over for the first while; thereafter you don't need to pay particularly close attention to it. 

Reviewer: ```suggestion
6. Simmer for around 45 minutes or until thick. Keep it at a medium heat, stirring occasionally. It shouldn't boil over, so you can leave it be.
```
